test/python/test_composite.py

```diff
@@ -16,16 +13,20 @@
     pass
 
 from cantera.composite import _pandas
-from . import utilities
 
+from .utilities import (
```

Comment:
My understanding is that idiomatic pytest code avoids imports from neighboring modules. I don't have a reference handy, but I'll try to dig one up.

Reply:
What's the alternative? Magic imports via fixtures defined in `conftest.py`?

Reply:
I believe `contest.py` is the standard approach, but I also have no strong opinions here.